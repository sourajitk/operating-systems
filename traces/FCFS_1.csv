1,0,1
2,0,3
3,0,2
4,7,1
5,7,3
6,7,2
