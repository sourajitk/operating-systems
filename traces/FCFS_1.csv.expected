1,1
2,4
3,6
4,8
5,11
6,13
